climakitae/data_loaders.py

```diff
@@ -215,14 +215,14 @@ def _process_and_concat(selections, location, dsets, cat_subset):
 
     if True in ["SSP" in one for one in selections.scenario_ssp]:
         if "Historical Climate" in selections.scenario_historical:
-            # Historical climate will be appended to the SSP data
-            append_historical = True
+            if selections.time_slice[0] > 2015:
```

Comment:
Just wondering why this is a `>`? Are we fixing a time slice filter earlier in the code and we set this `True` even though we don't actually append historical?

Reply:
`append_historical` used to be a param of DataSelectors, which is probably what you're thinking of. The code has since been restructured to automatically append historical if Historical Climate and an SSP is selected. The `append_historical` variable in this chunk of code is just used here in the backend data retrieval, not set by the user. 

The code now checks for time slice > 2015 because the Historical period runs until 2015, so if the time slice is 2016+, we don't need to append the historical period. 

Does that clear it up?

Reply:
I get it but just seems like the logic should be reversed?

Reply:
Yeah... you're right. Ugh. Just opened a new PR!